styleguide/_themes/derek/scss/components/_ceiling.scss

```diff
@@ -44,13 +46,11 @@ $ceiling-font-size: 11px;
     margin-bottom: 0;
 
     > li{
-      margin: $ceiling-margin 0;
-      padding: 0;
-      &:after {
-        content: "|";
-      }
-      &:last-child::after {
-        content: " ";
+      margin: floor($ceiling-margin + 2) 0;
```

Comment:
@WidgetsBurritos it's basically a just in case an integer gets changed or we're dividing by something that results in a non whole number. A safety net. @bighappyface what do you think? leave it in as a just in case or is it unnecessary?